index.html

```diff
@@ -1,13 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="container"></div>
-  </body>
-  <script src="script.js"></script>
-</html>
+    <head>
+        <meta charset="UTF-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Vanilla Todo</title>
+        <link rel="stylesheet" href="style.css"/>
+    </head>
+    <body>
+        <div class="container">
+            <div class="firstRow">
+                <h3>대기중 (
+                    <span class="firstCount">0</span>
+                     )
+                </h3>
+            </div>
+            <div class="firstList">
+            </div>
```

Comment:
class의 이름을  first 보다는 많은 list들이 나왔다고 봤을 때! 
어떤 list 인지 쉽게 찾기 위해 

```suggestion
            <div class="todo-list">
            </div>
```
이런식으로 바꿔주시면 좋지 않을까 생각이 듭니다!

Reply:
넵 이것도 제가 변수명을 너무 막 지었네요 ㅋㅋ